### DISCUSSION TITLE
Can i change superset logo without using docker

### DISCUSSION THREAD

**Discussion by Anuragpandey99x** [apache/superset#20028] [Q&A / Help] [answered]
## Screenshot

[drag & drop image(s) here!]

## Description

[describe the issue here!]

## Design input
[describe any input/collaboration you'd like from designers, and
tag accordingly. For design review, add the
label `design:review`. If this includes a design proposal,
include the label `design:suggest`]

**Comment by Anuragpandey99x** [comment]
@villebro @mistercrunch @betodealmeida @krishnap @kristw

**Comment by rusackas** [comment, marked as answer]
• The short answer is yes... you can change the URL path in `config.py`
• This is not really an issue with the software/codebase, so I'm going to move it to Github Discussions in case you have further questions about the details of configuration
• When filing issues, please make sure you're as descriptive as possible, so we can best address your concerns.

**Reply by Anuragpandey99x** [reply]
In which URL i have to make changes I did try to do the changes in APP_ICON it doesn't reflect at all.
